lib/Echidna/ABI.hs

```diff
@@ -182,8 +182,11 @@ dropBS b = Gen.choice [ BS.drop <$> Gen.element [1..BS.length b]   <*> pure b
                       , BS.take <$> Gen.element [0..BS.length b-1] <*> pure b
                       ]
 
-changeBS :: MonadGen m => ByteString -> m ByteString
-changeBS b = Gen.choice $ [changeChar, addBS, dropBS] <&> ($ b)
+changeDynamicBS :: MonadGen m => ByteString -> m ByteString
+changeDynamicBS b = Gen.choice $ [changeChar, addBS, dropBS] <&> ($ b)
+
+changeStaticBS :: MonadGen m => ByteString -> m ByteString
+changeStaticBS b = changeChar b
```

Comment:
I'm more inclined to just use `changeChar` rather than give it a second name here